pkg/webhook/webhook.go

```diff
@@ -453,6 +457,72 @@ func (wh *webhook) validatePVC(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRe
 	return response
 }
 
+func (wh *webhook) validateNamespace(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
+	req := ar.Request
+	response := &v1beta1.AdmissionResponse{}
+	response.Allowed = true
+	// validates only if requested operation is CREATE or DELETE
+	if req.Operation == v1beta1.Delete {
+		return wh.validateNamespaceDeleteRequest(req)
+	}
+	klog.V(2).Info("Admission wehbook for PVC module not " +
```

Comment:
Can you also check that this rule only applies to openebs namespace? Other namespace requests shouldn't get impacted by this. 

Add a test to verify that when BDCs are present, a test namespace can be created and deleted without any issues.

Reply:
Fixed the log message and added the manual test result to the PR description.